mutiny-core/src/nodemanager.rs

```diff
@@ -637,31 +614,34 @@ impl<S: MutinyStorage> NodeManager<S> {
 
         let gossip_sync = Arc::new(gossip_sync);
 
-        // load lsp clients, if any
-        let lsp_clients: Vec<LspClient> = match c.lsp_url.clone() {
+        // load voltage flow configs, if any
+        let mut lsp_configs: Vec<LspConfig> = match c.lsp_url.clone() {
             // check if string is some and not an empty string
             // and safe_mode is not enabled
-            Some(lsp_urls) if !lsp_urls.is_empty() && !c.safe_mode => {
-                let urls: Vec<&str> = lsp_urls.split(',').collect();
-
-                let futs = urls.into_iter().map(|url| LspClient::new(url.trim()));
-
-                let results = futures::future::join_all(futs).await;
-
-                results
-                    .into_iter()
-                    .flat_map(|res| match res {
-                        Ok(client) => Some(client),
-                        Err(e) => {
-                            log_warn!(logger, "Error starting up lsp client: {e}");
-                            None
-                        }
-                    })
-                    .collect()
-            }
+            Some(lsp_urls) if !lsp_urls.is_empty() && !c.safe_mode => lsp_urls
+                .split(',')
+                .map(|s| LspConfig::new_voltage_flow(s.trim().to_string()))
+                .collect(),
             _ => Vec::new(),
         };
 
+        // load lsps flow configs, if any
+        if let Some(lsp_connection_string) = c.lsp_connection_string.clone() {
```

Comment:
I wonder if we should remove the "multi lsp at the same time" option. It's really not practical to be interfacing with two at once. @benthecarman ?

Reply:
ack from me